goal_src/engine/gfx/generic/generic.gc

```diff
@@ -5,3 +5,150 @@
 ;; name in dgo: generic
 ;; dgos: GAME, ENGINE
 
+; This is actually defined in generic-vu1.gc but this file builds first
+(define-extern generic-init-buf (function dma-buffer int uint none))
```

Comment:
In general, when there's stuff like this I try to put them in the `-h.gc` files, kind of like you would with C `.h` files, instead of in the file its used in.  It's not a huge deal but I think it will keep things a little bit more organized.

So in this case, it could go in `generic-h.gc`.